hydraw/hydraw.cabal

```diff
@@ -1,6 +1,6 @@
 cabal-version: 2.2
 name:          hydraw
-version:       0.0.1
+version:       0.10.0
```

Comment:
I used the future version here since it seems like we would like to run hydraw on mainnet so might want to version it the same as other packages?

Reply:
I would not consider `hydraw` as part of the `hydra` delivery. Also, I would not do this within this PR. "Start versioning hydraw" should be a separate task (I believe the version it has should also be detached from hydra... -> separate repo?)